dashboard/v1.1/src/layouts/QuickInputFab.tsx

```diff
@@ -40,4 +45,4 @@ const QuickInputFab = () => {
   );
 };
 
-export default QuickInputFab;
+export default withRouter(QuickInputFab);
```

Comment:
Why do we need to wrap this component with `withRouter`? If it is to access `location` props inside this component, let's add this logic to some top level component which can pass these values to child components